desktop/assets/texts/es-ES/preferences_dialog.ftl

```diff
@@ -8,6 +8,9 @@ graphics-power-high = Alto (para gráficos dedicados)
 language = Idioma
 audio-output-device = Dispositivo de reproducción de audio
 audio-output-device-default = Predeterminado del sistema
+enable-openh264 = Habilitar Open H264
```

Comment:
@sombraguerrero I removed this space on CrowdIn to make it "OpenH264", which is the name of the library/project we're utilizing, so I figured it's best to keep it as is. I hope you don't mind.

Reply:
I don't even know why I put that space there, because I didn't in other places. :P